kolena/_utils/batched_load.py

```diff
@@ -79,6 +82,18 @@ def upload_data_frame_chunk(df_chunk: pd.DataFrame, load_uuid: str) -> None:
 DFType = TypeVar("DFType", bound=LoadableDataFrame)
 
 
+def upload_smart_chunk_data_frame(df: pd.DataFrame, uuid: str, rows: int = 1000) -> None:
+    batch_size = (BATCH_LIMIT // get_preflight_export_size(df, rows)) * rows
+    upload_data_frame(df, batch_size, uuid)
+
+
+def get_preflight_export_size(df: pd.DataFrame, rows: int) -> int:
```

Comment:
could we add `_` prefix for this func? b/c it should be only used in this file

Reply:
will add